eks/asg.go

```diff
@@ -107,9 +108,11 @@ func waitAndVerifyNewInstances(
 	}
 	elbs, err := kubectl.GetAWSLoadBalancers(kubectlOptions)
 	if err != nil {
-		logger.Errorf("Error retrieving associated ELB names of the Kubernetes services.")
-		logger.Errorf("Undo by terminating all the new instances and trying again")
-		return err
+		if !(ignoreLoadBalancerState) {
+			logger.Errorf("Error retrieving associated ELB names of the Kubernetes services.")
+			logger.Errorf("Undo by terminating all the new instances and trying again")
+			return err
+		}
 	}
 	err = waitForAnyInstancesRegisteredToELB(elbSvc, elbv2Svc, elbs, instanceIds)
```

Comment:
In case of an error, `elbs` here will always be `nil` so this will effectively be a no op: https://github.com/gruntwork-io/kubergrunt/blob/0920bce8179daeb93899770ec19732f9ddc42a91/kubectl/service.go#L73-L80  

I would rework `GetAWSLoadBalancers` to return any ELBs that it is able to along with the errors so that you can at least wait for instances to register to the load balancers that are in the proper state even when you have triggered this flag.